websocket.go

```diff
@@ -124,13 +138,16 @@ func (t wsHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 			err = rerr
 			break
 		}
+		rx += len(buf)
+		n++
```

Comment:
Ideally, we only want to do these if logging is needed. But it might be too much effort for very little gain.

Reply:
I think the `if` will take more time than the add here ;)

Reply:
yeah, `if` won't be useful. I was thinking more of a decorator pattern, but that's going to change everything.